online-shopping-website/src/Components/ProductDetails.js

```diff
@@ -0,0 +1,128 @@
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import Stack from '@mui/material/Stack';
+import AddIcon from '@mui/icons-material/Add';
+import RemoveIcon from '@mui/icons-material/Remove';
+import {useParams} from 'react-router-dom';
+import Products from '../TestValues.json';
+
+class ProductButtons extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            quantity: 1,
+            show: true,
+            max: 5,
+            min: 0
+        };
+    }
+
+    IncrementItem = () => {
+        if (this.state.quantity != 10) {
+            this.setState({
+                quantity: this.state.quantity + 1
+            });
+        }
+    }
+    DecreaseItem = () => {
+        if (this.state.quantity != 1) {
+            this.setState({quantity: this.state.quantity - 1});
+        }
+    }
+
+    UpdateValue = (e) => {
+        this.setState({quantity: e.target.value});
+    }
+
+    get quantity() {
+        return this.state.quantity;
+    }
+
+    render() {
+        return (
+            <div className="ProductDetails-QuantityButtonsContainer">
+                <h3>Quantity</h3>
+                <Stack className="ProductDetails-QuantityButtonsStack" direction="row" spacing={1}>
+                    <Button className="ProductDetails-QuantityButtons" variant="contained"
+                            disabled={this.state.quantity == 10}
+                            onClick={this.IncrementItem}>
+                        <AddIcon/>
+                    </Button>
+                    <input className="inputne" value={this.state.quantity} onChange={this.UpdateValue}/>
+                    <Button className="ProductDetails-QuantityButtons" variant="contained"
+                            disabled={this.state.quantity == 1}
+                            onClick={this.DecreaseItem}>
+                        <RemoveIcon/>
+                    </Button>
+                </Stack>
```

Comment:
Can the + button be to the right of the quantity instead of the left please? It's just more standard UX

Reply:
Yup, makes more sense there